src/styles/creators.scss

```diff
@@ -0,0 +1,114 @@
+.creators {
+  margin-bottom: 100px;
+  position: relative;
+
+  @include tablet {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 30px;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    margin-bottom: 200px;
+    grid-gap: 120px;
+
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+
+    @include desktop {
+      padding-left: 100px;
+    }
+  }
+
+  &__title {
+    max-width: 280px;
+    position: relative;
+
+    @include tablet {
+      max-width: 100%;
+    }
+
```

Comment:
The max-width of the '.creators__title' class is set to 280px for both mobile and desktop versions. This is unnecessary repetition. If the value doesn't change on different screens, you can set it outside media queries.